linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_controller.js

```diff
@@ -81,11 +81,13 @@ var RouterController = (function () {
 
     _.each(routerContainerEls, function(container, router) {
       var $summaryEl = $(container.find(".summary")[0]);
-      var $serverEl = $(container.find(".server")[0]);
+      var $serverEl = $(container.find(".servers")[0]);
```

Comment:
nit: should this be `serversEl` (plural) now?  we have `$clientsEl` -- would be nice to keep them consistent